app/javascript/controllers/modal_controller.js

```diff
@@ -0,0 +1,19 @@
+import { Controller } from "@hotwired/stimulus"
+
+// Connects to data-controller="modal"
+export default class extends Controller {
+  static targets = [ "modal", "overlay" ]
+  connect() {
+    console.log("hello");
+  }
+
+  open() {
+    this.modalTarget.classList.remove('d-none');
+    this.overlayTarget.classList.add('overlay');
+  }
+
+  close() {
+    this.modalTarget.classList.add('d-none');
+    this.overlayTarget.classList.remove('overlay');
+  }
+}
```

Comment:
This looks good and clean, but let's maybe get rid of the logger, as it will clog up our console if we leave them in for everything 👻